scripts/run_tests

```diff
@@ -2,7 +2,14 @@
 import sys, os
 
 MOOSE_DIR = os.environ.get('MOOSE_DIR', os.path.abspath(os.path.join(os.path.dirname(os.path.realpath(__file__)), '..')))
-sys.path.append(os.path.join(MOOSE_DIR, 'python'))
 
-from TestHarness import TestHarness
+MOOSETOOLS_DIR = os.getenv('MOOSETOOLS_DIR', None)
+if MOOSETOOLS_DIR is not None:
+    print("Testing with 'moosetools' from {}".format(MOOSETOOLS_DIR))
```

Comment:
This is gonna be removed in the future, right? Probably worth making an issue when possible

Reply:
What is wrong with #17006?